Source/ToolCore/JSBind/JSBHeaderVisitor.h

```diff
@@ -284,6 +284,9 @@ class JSBHeaderVisitor : public SymbolVisitor
         if (function->isStatic())
             jfunction->SetStatic(true);
 
+        if (!function->isPublic())
+            jfunction->SetSkip(true);
+
```

Comment:
This goes along with the comment about processing private methods, that is an early out which should be kept instead of setting it to skip here...